src/xenia/patcher/plugin_loader.cc

```diff
@@ -49,7 +49,7 @@ void PluginLoader::LoadConfigs() {
 
   for (const auto& entry : dir_files) {
     const uint32_t title_id =
-        std::stoi(entry.name.filename().string(), nullptr, 16);
+        std::stoul(entry.name.filename().string(), nullptr, 16);
```

Comment:
To be fair I think it can be replaced by: ``string_util::from_string(entry.name.filename().string(), true);``